### PR TITLE
Add Storm service permissions to the declared permissions (SYN-7014)

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: synapse/lib/stormtypes.py [hunk 1]
```diff
@@ -907,6 +907,20 @@ class LibService(Lib):
...
+    _storm_lib_perms = (
+        {'perm': ('service', 'add'), 'gate': 'cortex',
+            'desc': 'Controls the ability to add a Storm Service to the Cortex.'},
+        {'perm': ('service', 'del'), 'gate': 'cortex',
+            'desc': 'Controls the ability to delete a Storm Service from the Cortex'},
+        {'perm': ('service', 'get'), 'gate': 'cortex',
+            'desc': 'Controls the ability to get the service object for any Storm service.'},
+        {'perm': ('service', 'get', '<iden>'), 'gate': 'cortex',
+            'desc': 'Controls the ability to get the service object for a Storm service by iden.'},
+        {'perm': ('service', 'get', '<name>'), 'gate': 'cortex',
```
Comment (invisig0th) [inline synapse/lib/stormtypes.py:919]: Do we still reference them by name or are they mostly by `<iden>` now? Names seem more fragile.

Comment (vEpiphyte) [reply synapse/lib/stormtypes.py:919]: technically the name permissions have been deprecated. I'll remove them.

File: synapse/lib/stormtypes.py [hunk 2]
```diff
@@ -907,6 +907,18 @@ class LibService(Lib):
                   'returns': {'type': 'boolean', 'desc': 'Returns true if the service is available, false on a '
                                                          'timeout waiting for the service to be ready.', }}},
     )
+    _storm_lib_perms = (
+        {'perm': ('service', 'add'), 'gate': 'cortex',
+            'desc': 'Controls the ability to add a Storm Service to the Cortex.'},
+        {'perm': ('service', 'del'), 'gate': 'cortex',
+            'desc': 'Controls the ability to delete a Storm Service from the Cortex'},
+        {'perm': ('service', 'get'), 'gate': 'cortex',
+            'desc': 'Controls the ability to get the service object for any Storm service.'},
```
Comment (Cisphyx) [inline synapse/lib/stormtypes.py:916]:
```suggestion
            'desc': 'Controls the ability to get the Service object for any Storm Service.'},
```

File: synapse/lib/stormtypes.py [hunk 3]
```diff
@@ -907,6 +907,18 @@ class LibService(Lib):
...
+        {'perm': ('service', 'get', '<iden>'), 'gate': 'cortex',
+            'desc': 'Controls the ability to get the service object for a Storm service by iden.'},
```
Comment (Cisphyx) [inline synapse/lib/stormtypes.py:918]:
```suggestion
            'desc': 'Controls the ability to get the Service object for a Storm Service by iden.'},
```

File: synapse/lib/stormtypes.py [hunk 4]
```diff
@@ -907,6 +907,18 @@ class LibService(Lib):
...
+        {'perm': ('service', 'list'), 'gate': 'cortex',
+         'desc': 'Controls the ability to list all available Storm services and their service definitions.'},
```
Comment (Cisphyx) [inline synapse/lib/stormtypes.py:920]:
```suggestion
         'desc': 'Controls the ability to list all available Storm Services and their service definitions.'},
```

**Review by Cisphyx — APPROVED** [review]

**Review by invisig0th — APPROVED** [review]

**Pull request merged**